web/src/features/fba/components/viz/FuelSummary.tsx

```diff
@@ -57,20 +57,32 @@ const columns: GridColDef[] = [
     renderCell: (params: GridRenderCellParams) => {
       return <FuelDistribution code={params.row['code']} percent={params.row['percent']} />
     }
+  },
+  {
+    field: 'criticalHours',
+    flex: 3,
+    headerClassName: 'fuel-summary-header',
+    headerName: 'Critical Hours',
+    minWidth: 120,
```

Comment:
I'm seeing a horizontal scroll bar for the fuel summary table which would be nice to get rid of. I played around a little and we could try:
1. Remove `flex: 3` and `width: 120` and add `minWidth: 80` for the code col def
2. Remove `minWidth: 200` and add `flex: 1` for the area col def
3. Remove `flex: 3` and add `minWidth: 110` for critical hours col def

Reply:
Nice! Yeah works locally for me, added changes here: https://github.com/bcgov/wps/pull/3909/commits/79e4d2937495c714b45050fecf6cbf654a3b68a1